gateway/living_world.py

```
@@ -0,0 +1,115 @@
+        # transport without importing the mycelium+aiohttp stack.
+        msg = {"from": self.self.name(), "presence": "awake", "kind": "presence"}
+        if publish is not None:
+            try:
+                publish(msg)
+            except Exception:
+                pass
+        else:
+            try:
+                asyncio.run(_announce_presence(self.self.name(), "awake"))
+            except Exception:
+                pass  # mycelium is eventually consistent; silence is fine
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)